test/dashdb.replaceById.test.js

```diff
@@ -0,0 +1,114 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-connector-dashdb
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+/* eslint-env node, mocha */
+process.env.NODE_ENV = 'test';
+
+require('./init.js');
+const assert = require('assert');
+
+let id, db, Book;
+const BOOK_TITLE = 'Rocky I';
+const REVISED_BOOK_TITLE = 'Rocky II';
+
+/*
+   This test suite is to test the changes surrounding the fix
+   in loopback-ibmdb related to issue :
+   https://github.com/strongloop/loopback-ibmdb/issues/79
+*/
+
+describe('replaceById with BOOK_ID', function() {
```

Comment:
If people don't know the background of this issue, I feel it's hard for them to understand what is the difference of "BOOK_ID" and "book_id" and why there are 2 tests.

Maybe rephrase the test title to be:
'replaceById with custom column name for id'